gitbook-docs/data_model.md

```diff
@@ -19,19 +19,21 @@ The simplest format is the full format, which is the complete Signal K data mode
       "navigation": {
         "speedOverGround": {
           "value": 4.32693662,
-          "$source": "0183./dev/ttyUSB0.GP.RMC",
+          "$source": "0183.ttyUSB0.GP.RMC",
```

Comment:
This is not how the Node server works right now. I assume in this example 0183 is the label, but the ttyUSB0 does not belong in the server's internal, symbolic reference.

For NMEA0183 data there are just two hierarchy levels, the first for label and the second one for talker. A single nmea0183 input, identified by label, may have data from several talkers, identified by talker id. A talker has a list of sentences it produces with timestamps for the last received ones:

![image](https://user-images.githubusercontent.com/1049678/32418610-f7cebea4-c275-11e7-9b8f-c07314e9eb0d.png)

(Dunno what's wrong with SK talker id, that is probably an error in the new 0183 parser, need to check that.)